main.go

```diff
@@ -64,16 +66,23 @@ func getResources(state string) (resources []string) {
 }
 
 func processResources(state string, resources []string) (b2 []byte) {
+
 	var conditions []interface{}
 	for _, resource := range resources {
 		if strings.Contains(resource, "aws_instance") {
-			thing := Server{}
-			conditions = append(conditions, thing.Process(state, resource)...)
+
+			server := Server{}
+			conditions = append(conditions, server.Process(state, resource)...)
+
+			app := App{}
+			conditions = append(conditions, app.Process(state, resource)...)
+
 		} else if strings.Contains(resource, "aws_sqs_queue") {
-			thing := SQS{}
-			conditions = append(conditions, thing.Process(state, resource)...)
+			sqs := SQS{}
+			conditions = append(conditions, sqs.Process(state, resource)...)
 		}
 	}
+
```

Comment:
whitespace :)
sorry if that was intentional

Reply:
gofmt force to leave the space at the end of the file.